main.go

```diff
@@ -90,12 +90,13 @@ func main() {
 		setupLog.Error(err, "unable to get config")
 		os.Exit(1)
 	}
-
+	logOpts := []logr.Option{logr.WithComponentName("vcluster-controller")}
+	log, err := logr.NewLoggerWithOptions(logOpts...)
```

Comment:
You can write that as:
```go
	log, err := logr.NewLoggerWithOptions(logr.WithComponentName("vcluster-controller"))
```

but I'd recommend also using the env vars for this, so more in the likes of:

```suggestion
	log, err := logr.NewLoggerWithOptions(
	  logr.WithOptionsFromEnv(),
	  logr.WithComponentName("vcluster-controller"),
	)
```